plugin.sh

```diff
@@ -1,11 +1,32 @@
-#!/busybox/sh
+#!/busybox/busybox sh
```

Comment:
Was the exec bit intentionally removed?

Reply:
I don’t have removed the exec bit, I used this shell to help shellcheck to identify correctly the shell features